server/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -174,7 +175,7 @@ public ActionRequestValidationException validate() {
                 return validationException;
             }
 
-            if (ifSeqNo != SequenceNumbers.UNASSIGNED_SEQ_NO || ifPrimaryTerm != 0) {
+            if (ifSeqNo != UNASSIGNED_SEQ_NO || ifPrimaryTerm != 0) {
```

Comment:
there's another 0 here

Reply:
I missed that pattern. Getting one more regex out of my toolbox.